### ISSUE TITLE
Banner causing issue on Macupdate.com

### ISSUE THREAD

**Issue by nadesco** [gorhill/uBlock#2028]
### Describe the issue

Can’t completely block one particular banner near the top of the page, the one with the rotating ads and 4 blue spots (slider controls?) just under it. 

### One or more specific URLs where the issue occurs

https://www.macupdate.com/

### Screenshot in which the issue can be seen

![frameonblank](https://cloud.githubusercontent.com/assets/20380923/18814576/91a3c1f0-8352-11e6-9bc5-c160c9a8e515.jpg)
![frameoverlaid](https://cloud.githubusercontent.com/assets/20380923/18814577/91a7d510-8352-11e6-9206-34f2e3626026.jpg)

### Steps for anyone to reproduce the issue

1) add custom filters using the contextual menu - one for each of the ads, one for the sliders and one for the frame/banner itself.

2) toggling cosmetic filtering also has an effect (currently default) but doesn't fully block it.

Expected result: the ad would disappear.

Actual result: I’ve managed to partially block it. Only a thin black frame remains on a blank white area. If I add a filter to block the frame the blank area disappears and the frame drops down over the update links etc. The blanked out frame also toggles on and off, as though it’s still running its invisible ads.

### Your settings

- Browser/version: Firefox V. 50 beta 1.
- uBlock Origin version: 1.9.8

##### Your filter lists

Default filter lists

##### Your custom filters (if any)
As below ...

! 25/09/2016, 18:39:01 https://www.macupdate.com/
||dl2.macupdate.com/images/app_billboards/Ramme.jpg$image

! 25/09/2016, 18:39:11 https://www.macupdate.com/
||dl2.macupdate.com/images/app_billboards/FallBundleBillboard2016.jpg$image

! 25/09/2016, 18:39:22 https://www.macupdate.com/
||dl2.macupdate.com/images/app_billboards/DeskDock.jpg$image

! 25/09/2016, 18:39:35 https://www.macupdate.com/
||dl2.macupdate.com/images/app_billboards/Billboard-Appcollection-Tunabelly.jpg$image

! 25/09/2016, 18:39:51 https://www.macupdate.com/
www.macupdate.com###billboard_a-better-finder-rename

! 25/09/2016, 18:40:01 https://www.macupdate.com/
www.macupdate.com###nfmlfas-slider-controls

! 25/09/2016, 18:41:17 https://www.macupdate.com/
www.macupdate.com###billboard_deskdock-server

! 25/09/2016, 18:41:54 https://www.macupdate.com/
www.macupdate.com###billboard_deskdock-server

! 25/09/2016, 18:46:26 https://www.macupdate.com/
www.macupdate.com###nfml-featuredapp-slider > div:nth-of-type(2)

**Comment by nadesco** [comment]
Forgot to mention that I disabled all firefox addons except for uBlock, made no difference though.

**Comment by gwarser** [comment]
Try this: `www.macupdate.com###nfml-featuredapp-slider`

**Comment by nadesco** [comment]
Thanks. I copied and pasted it into Temporary rules, clicked the save button but it was discarded. Tried again, same thing.

**Comment by Hrxn** [comment]
Paste into your filter list?

**Comment by gorhill** [comment]
That billboard is made of many images, if you click _Pick_ again after selecting one and select a new image, uBO will offer you the filter `||dl2.macupdate.com/images/app_billboards/*.jpg` as a choice. Select this filter, and click _Create_.

Now there is a placeholder left behind, because of how the site is designed. Select that placeholder with element picker, and uBO will presents a list of choice, from narrowest to broadest. You do not have to select the first one, find the one that accomplishes best what you want. Try selecting different choices starting from the top, you will find that the entry `###nfml-featuredapp-slider` seems to work best, click _Preview_ to confirm, click _Create_.

This way I created the two filters (delete all the other filters you tried to create for that one element):
- `||dl2.macupdate.com/images/app_billboards/*.jpg$image`
- `www.macupdate.com###nfml-featuredapp-slider`

So this is a filter list issue (should not have been opened here), and a "didn't read the doc issue", I repeated above what is already spelled out [in the doc](https://github.com/gorhill/uBlock/wiki/Element-picker).
